### DISCUSSION TITLE
Arm64 support for standalone systems

### DISCUSSION THREAD

**Discussion by kraza8** [milvus-io/milvus#30345] [Q&A and General discussion] [answered]
As per this link, Milvus 2.3 supports ARM64:
https://milvus.io/blog/unveiling-milvus-2-3-milestone-release-offering-support-for-gpu-arm64-cdc-and-other-features.md 

But this other link says that only x86 & apple silicon are supported:
https://milvus.io/docs/prerequisite-docker.md

Can someone please clarify this discrepancy?

Thanks

**Comment by yhmo** [comment, marked as answer]
The Milvus server itself can support arm64 now. You can deploy milvus standalone on arm64 machine.
But the dependent Pulsar v2.8.2 doesn't support arm64. So, milvus cluster will fail on arm64 machine. The Pulsar v3.0 can support arm64, once we successfully upgrade pulsar to v3.0 for milvus, we will add the arm64 support on the prerequisite page.

**Reply by kraza8** [reply]
Thank you. We want to use Milvus as a standalone on arm64 without clustering.
